src/markdown-to-html.pipe.ts

```diff
@@ -6,9 +6,9 @@ import * as marked from 'marked';
 })
 
 export class MarkdownToHtmlPipe implements PipeTransform {
-    public transform(markdown: string, options?: MarkedOptions): string {
+    public transform(markdown: string, options?: MarkedOptions, inline?: boolean): string {
         if (markdown == null) return '';
-        return marked(markdown, options);
+        return (inline) ? marked.inlineLexer(markdown, [], options) : marked(markdown, options);
```

Comment:
Thanks for your contribution!
Could you wrap `inline` into an object?

It think `{ inline: true }` is much more expressive
`[innerHTML]="t.text || t.full_text | MarkdownToHtml:{}:{inline: true}"`